C7/MainMenuTheme.tres

```diff
@@ -0,0 +1,32 @@
+[gd_resource type="Theme" load_steps=6 format=2]
```

Comment:
I copied our main theme so I could change the colors on the buttons on the main menu, which shouldn't necessarily follow the main theme.  We probably could go back to the bad old days of font color overrides everywhere, and that would avoid duplicating this... but I'm not sure it's better.